tests/tck/features/match/SelfReflectiveEdges.feature

```diff
@@ -28,13 +42,4 @@ Feature: Matches on self-reflective edges
     Then the result should be, in any order:
       | e1                                                                 | e2                                                                 |
       | [:teammate "Hades"->"Hades" @0 {end_year: 3000, start_year: 3000}] | [:like "Hades"->"Hades" @0 {likeness: 3000}]                       |
-      | [:like "Hades"->"Hades" @0 {likeness: 3000}]                       | [:teammate "Hades"->"Hades" @0 {end_year: 3000, start_year: 3000}] |
-    When executing query:
-      """
-      DELETE EDGE serve "Hades"->"Underworld";
-      DELETE EDGE teammate "Hades"->"Hades";
-      DELETE EDGE like "Hades"->"Hades";
-      DELETE VERTEX "Underworld";
-      DELETE VERTEX "Hades";
-      """
-    Then the execution should be successful
+      | [:like "Hades"->"Hades" @0 {likeness: 3000}]                       | [:teammate "Hades"->"Hades" @0 {end_year: 3000, start_year: 3000}] |
```

Comment:
could use the following step to delete the created space after finishing the scenario:

```
And drop the used space
```

Reply:
Thanks for the advice. Fixed.